core/schema/sdk.go

```diff
@@ -276,8 +276,8 @@ func (sdk *moduleSDK) GenerateClient(
 				Value: dagql.String(outputDir),
 			},
 			{
-				Name:  "useLocalSdk",
-				Value: dagql.NewBoolean(useLocalSDK),
+				Name:  "dev",
+				Value: dagql.NewBoolean(dev),
```

Comment:
:eyes: Bit confused here, is this not going to break with SDKs that don't have a `dev` argument? Why does not work at all, I don't see any changes to any other runtimes for this, like Python.

Is the `dev` argument not getting sent if it's `false`? What happens if I initialize it with `--dev=true` for `--sdk=python`? We should only attach this if the SDK supports it - we can do introspection, similar to how @rajatjindal is doing in https://github.com/dagger/dagger/pull/9323/files#diff-0f56671aa816ba46aa7ca99e164f99cb39d656cea7c4293b9b475b30030ea8eeR247.

If it's not defined - we should either: 1. error out nicely, 2. ignore it, and discard the dev option. Definitely not throw an internal error out to the user.